src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -179,6 +179,10 @@ define(['sulucontent/components/content/preview/main'], function(Preview) {
             this.createForm(this.data).then(function() {
                 this.initializeResourceLocator();
                 this.changeTemplateDropdownHandler();
+
+                if (!!Config.get('sulu-content').collaboration.enabled) {
```

Comment:
Well, if I don't want to have the collaboration stuff, I simply don't include that bundle, then there is also nothing to handle :wink: